src/components/InputTodo.js

```diff
@@ -0,0 +1,5 @@
+const InputTodo = () => (
+  <div>input field here...</div>
```

Comment:
The code looks fine😎, but it doesn't contain any actual input field. It only returns a <div> element with some text You should modify it to include an actual input element, for example:  
```suggestion
  <div>
    <input type="text" placeholder="Enter a task" />
  </div>
```

Reply:
Alright, I will fix that.

Thanks for pointing it out.